test/unit/model/consent.test.ts

```diff
@@ -315,53 +333,55 @@ describe('src/model/consent', (): void => {
       expect(consents.length).toEqual(0)
     })
 
-    it('throws an error for a non-existent consent', async (): Promise<void> => {
+    it('throws an error for a non-existent consent',
+      async (): Promise<void> => {
       // Action
-      await expect(consentDB.delete(completeConsent.id))
-        .rejects.toThrowError(NotFoundError)
-    })
+        await expect(consentDB.delete(completeConsent.id))
+          .rejects.toThrowError(NotFoundError)
+      })
 
-    it('deletes associated scopes on deleting a consent', async (): Promise<void> => {
+    it('deletes associated scopes on deleting a consent',
+      async (): Promise<void> => {
       // Setup
-      await Db<Consent>('Consent')
-        .insert(completeConsent)
+        await Db<Consent>('Consent')
+          .insert(completeConsent)
 
-      // Pre action Assertion
-      await Db('Scope')
-        .insert({
-          consentId: '1234',
-          action: 'accounts.transfer',
-          accountId: '78901-12345'
-        })
+        // Pre action Assertion
+        await Db('Scope')
+          .insert({
+            consentId: '1234',
+            action: 'accounts.transfer',
+            accountId: '78901-12345'
+          })
 
-      await Db('Scope')
-        .insert({
-          consentId: '1234',
-          action: 'accounts.balance',
-          accountId: '38383-22992'
-        })
+        await Db('Scope')
+          .insert({
+            consentId: '1234',
+            action: 'accounts.balance',
+            accountId: '38383-22992'
+          })
 
-      let scopes = await Db('Scope')
-        .select('*')
-        .where({ consentId: completeConsent.id })
+        let scopes = await Db('Scope')
+          .select('*')
+          .where({ consentId: completeConsent.id })
 
-      expect(scopes.length).toEqual(2)
+        expect(scopes.length).toEqual(2)
 
-      // Action
-      const deleteCount: number = await consentDB.delete(completeConsent.id)
+        // Action
+        const deleteCount: number = await consentDB.delete(completeConsent.id)
 
-      expect(deleteCount).toEqual(1)
+        expect(deleteCount).toEqual(1)
 
-      scopes = await Db('Scope')
-        .select('*')
-        .where({ consentId: completeConsent.id })
+        scopes = await Db('Scope')
+          .select('*')
+          .where({ consentId: completeConsent.id })
 
-      const consents = await Db('Consent')
-        .select('*')
-        .where({ id: completeConsent.id })
+        const consents = await Db('Consent')
+          .select('*')
+          .where({ id: completeConsent.id })
 
-      expect(consents.length).toEqual(0)
-      expect(scopes.length).toEqual(0)
-    })
+        expect(consents.length).toEqual(0)
+        expect(scopes.length).toEqual(0)
+      })
   })
 })
```

Comment:
Add unit tests for correct insert/update behavior of `revokedAt`.

Reply:
Separation of concerns - will make a separate PR for further testing